### ISSUE TITLE
`pixi info` reports version `0.1.0` instead of `0.27.0`

### ISSUE THREAD

**Issue by imagejan** [prefix-dev/pixi#1758]
### Checks

- [X] I have checked that this issue has not already been reported.

- [X] I have confirmed this bug exists on the [latest version](https://github.com/prefix-dev/pixi/releases) of pixi, using `pixi --version`.

### Reproducible example

```shell
pixi info
```

### Issue description

The first line of output (independent of inside or outside a project directory) will show:

```
      Pixi version: 0.1.0
```

### Expected behavior

With the latest version, the output should be:
```
      Pixi version: 0.27.0
```

It worked correctly with version `0.26.0`.

**Comment by wolfv** [comment]
Can confirm!

**Comment by ruben-arts** [comment]
Fixed by #1759
